File: 1051-height-checker/1051-height-checker.java
class Solution {
    public int heightChecker(int[] heights) {
        int[] sorted = heights.clone();
        int diff = 0;
        Arrays.sort(sorted);
        
        for(int i = 0; i < heights.length; i++){
            if(heights[i] != sorted[i])
                diff++;
        }
        
        return diff;
    }
}

File: 1051-height-checker/1051-height-checker.swift
class Solution {
    func heightChecker(_ heights: [Int]) -> Int {
        var expected = heights.sorted(by: <)
        var cnt = 0
        
        var i = 0 
        while i < heights.count {
            if heights[i] != expected[i] { cnt += 1 }
            i += 1
        }
        
        return cnt
    }
}